control_panel_api/permissions.py

```diff
@@ -91,6 +91,26 @@ def has_object_permission(self, request, view, obj):
 
         return request.user == obj
 
+# NOTE: James is doing some work related to this class as well,
+#       that's why I'm not testing it - he'll add tests.
+#       Only thing relevant for filtering work is `list`/`retrive` actions
```

Comment:
Don't put notes like this in the code. It doesn't document the code so it doesn't belong here. Add a comment in the PR instead.